supervisord.conf

```diff
@@ -161,9 +161,9 @@ autorestart = true
 priority = 13
 command =
     /usr/sbin/pmm-managed
-        --prometheus-config /etc/prometheus.yml
-        --prometheus-url http://127.0.0.1:9090/prometheus
+        -database /srv/pmm-managed/pmm-managed.sqlite3
+        -prometheus-config /etc/prometheus.yml
+        -prometheus-url http://127.0.0.1:9090/prometheus
```

Comment:
We use stdlib `flag` package which uses `-` for flags. `--` works, but not documented to work. Let's play safe.